numpy/lib/index_tricks.py

```diff
@@ -840,6 +840,26 @@ def fill_diagonal(a, val, wrap=False):
            [0, 4, 0, 0, 0],
            [0, 0, 4, 0, 0]])
 
+    Anti-diagonal example:
+    The anti-diagonal can be obtained by reversing the order of elements
+    using either `numpy.flipud` or `numpy.fliplr`.
+
+    >>> a = np.zeros((3,3))
+    >>> a
+    array([[0., 0., 0.],
+           [0., 0., 0.],
+           [0., 0., 0.]])
```

Comment:
No need to show the output of zeros here, I'd argue it's obvious and just a distraction